apps/frontend/src/components/global/upload_tabs/FileReader.vue

```diff
@@ -27,6 +27,7 @@
             <li>Static Analysis Results Interchange Format (SARIF)</li>
             <li>Scoutsuite</li>
             <li>Snyk</li>
+            <li>Tenable (Nessus)</li>
```

Comment:
Tenable.sc API?

Reply:
@ejaronne suggested that we do not use the .sc here